test/index.test.ts

```diff
@@ -293,7 +405,191 @@ describe('HipThrusTS', () => {
     // @todo: add test coverage for TRIPLE-OPERATOR pipes
     // @todo: add test coverage for QUADRUPLE-AND-MORE-OPERATOR pipes (use empty objects)
     // @todo: add test coverage for multi-stage operators
+    describe('HTPipeTest', () => {
```

Comment:
I'd make attachData's return value actually use its input so we're sure there's no disconnection... so return something like `{c: cReturned, b: bReturned, aout: context.a }` instead
- [ ] validate wiring

Reply:
done